include/libopencm3/stm32/f4/dac.h

```diff
@@ -33,5 +33,21 @@ LGPL License Terms @ref lgpl_license
 
 #include <libopencm3/stm32/common/dac_common_all.h>
 
+
+/* DAC status register (DAC_SR) */
+/* not available on STM32F4x1/STM32F4x2 */
+#define DAC_SR				MMIO32(DAC_BASE + 0x34)
+
+/* --- DAC_SR values ------------------------------------------------------- */
+
+/* DMAUDR2: DAC channel 1 DMA underrun flag */
```

Comment:
I actually based it on `dac_common_all.h`, where all the comments like this are with just one `*`. Is it wrong there as well? (In case I can modify those too)

Reply:
no, don't go and modify anything else, we're just in a "moving forwards" style.  updating exsting stuff starts becoming feature creep and hard to review when it's part of existing stuff.